src/Components/Patient/DailyRounds.tsx

```diff
@@ -139,50 +132,55 @@ export const DailyRounds = (props: any) => {
     "rhythm_detail",
     "consciousness_level",
   ];
-
   const fetchRoundDetails = useCallback(
     async (status: statusType) => {
```

Comment:
need not wrap it in useCallback with abort status as params when using useQuery.

Reply:
@rithviknishad 
Removing the useCallback is affecting the input fields because they are not receiving the exact data as expected. There are multiple GET requests, each modifying the same data. I am using requests instead of useQuery for fetching the data. I referred to the patientRegister file for the same.

Should I still consider using useQuery?

Reply:
Okay, then let's not touch the state management if it works 😄